monitoring/mock_uss/msgsigning/routes_msgsigning.py

```diff
@@ -0,0 +1,31 @@
+import flask
+
+from monitoring.mock_uss import webapp, config
+from monitoring.mock_uss.msgsigning.database import db
+from loguru import logger
+import os
+
+
+@webapp.route(
+    "/mock/msgsigning/.well-known/uas-traffic-management/pub.der", methods=["GET"]
+)
+def get_public_key():
+    public_key_file_location = os.path.join(
+        webapp.config.get(config.KEY_CERT_BASE_PATH), db.value.public_key_name
+    )
+
+    logger.info("Retrieving public key file from {}".format(public_key_file_location))
+
+    return flask.send_file(public_key_file_location)
+
+
+@webapp.route("/mock/msgsigning/set-keypair", methods=["POST"])
+def set_keypair():
+    # TODO: Update keypair to use via a received JSON object -> {"public_key": "/path", "private_key": "/path"}
+    return flask.jsonify({"message": "Not yet implemented"}), 501
+
```

Comment:
General question, would having an endpoint to dynamically alter the keypair mean a change in the [uas_standards](https://github.com/interuss/uas_standards) project? Something like `KeyPairRequest`?

Reply:
We don't currently have a standard interface for the mock_uss -- for instance, the [SP](https://github.com/interuss/dss/blob/master/monitoring/mock_uss/ridsp/routes_behavior.py) and [DP](https://github.com/interuss/dss/blob/master/monitoring/mock_uss/riddp/routes_behavior.py) behavior endpoints are not listed in any interfaces since they're only used internally within InterUSS tools.  Since mock_uss is the only USS that would need to swap out certificates, I don't think we would need to communicate this interface outside the InterUSS domain (so, same category as those behavior endpoints linked above).  When a test scenario requires a [MockUSSResource](https://github.com/interuss/dss/blob/master/monitoring/uss_qualifier/resources/interuss/mock_uss.py), the implied contract in providing that resource is that the referenced server actually is a current instance of mock_uss, so we can assume we (uss_qualifier) can use any of the mock_uss endpoints there regardless of whether they're published in YAML somewhere.  So, all I think we need to do to support dynamic mock_uss keypairs is to add a simple endpoint that will look a lot like the behavior endpoints linked above.  uss_qualifier can then use that endpoint when it is provided a MockUSSResource.  [Small PRs are good](https://github.com/interuss/dss/blob/master/CONTRIBUTING.md#general-principles) so I'd generally recommend resisting the urge to tack on additional functionality to a PR (saving it for the next PR instead), but I don't think it would be completely unreasonable for this PR to include that functionality.  Again though, smaller PRs are better, so saving that until a follow-up PR would be my personal recommendation.

[The automated_testing_interfaces repository](https://github.com/interuss/automated_testing_interfaces) contains the interfaces InterUSS created that USSs need to implement in order to be tested in certain ways by InterUSS tools -- so, for instance, that's where the `scd.yaml` API that flight planner USSs need to implement is canonically located.  I don't think a USS under test would need to support the ability to dynamically alter its keypair to be effectively tested by InterUSS, so we probably wouldn't add an endpoint/capability to support that in `scd.yaml`, but of course we're open to arguments/explanations otherwise.

[The uas_standards repository](https://github.com/interuss/uas_standards) just contains handy tools for working with standard interfaces -- none of the interfaces are actually defined there.  The majority of library content in that repository is code auto-generated from the source interface.

Reply:
Thanks for the explanation! Are we good to keep those stub endpoints, then in the future PR come back and actually implement? I can take out for now and just add all at once later, let me know what you'd prefer

Reply:
Thanks for the explanation!